sed/calibrator/delay.py

```diff
@@ -92,8 +109,7 @@ def append_delay_axis(
                 Defaults to config["delay"]["p2_key"]
             t0_key (str, optional): hdf5 key for t0 value (mm).
                 Defaults to config["delay"]["t0_key"]
-            verbose (bool, optional): Option to print out diagnostic information.
-                Defaults to True.
+            suppress_output (bool, optional): Option to suppress log output. Defaults to False.
```

Comment:
Why is suppress output flag needed? If it's set to True, not even at debug level, this information will be visible.
I'd say we can set all print to logger.info like you did without the `not suppress_output` conditional. Then, if user selects verbose, it prints out the information (already done). And for debug, it uses what I wrote in another comment.

Reply:
This is used for the timed dataframes. If they are present, each output appears twice otherwise.